### PR TITLE
Add unmerged lora integration test

### PR DESCRIPTION
**Pull request by rohithkrn**

### CODE REVIEW

File: .github/workflows/llm_integration.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ jobs:
         with:
           python-version: '3.10.x'
       - name: Install pip dependencies
-        run: pip3 install requests numpy
+        run: pip3 install requests numpy huggingface_hub
```
Comment (lanking520) [inline .github/workflows/llm_integration.yml:130]: why we need to install this?

**Review by lanking520 — COMMENTED** [review]
since all changes are in the testing code, could you push to a DJLServing repo branch and show a dryrun with pipeline to see if test passed?

File: .github/workflows/llm_integration.yml (hunk 1 above)
Comment (rohithkrn) [reply .github/workflows/llm_integration.yml:130]: I am using `snapshot_download` function to download adapters on the fly.
https://github.com/deepjavalibrary/djl-serving/pull/1110/files#diff-1348e5a6c0a696af2fa5904cc01e2bd27c060d427dcf3d30997f8f53201c74ccR490

**Comment by rohithkrn** [comment]
I did run a test log - https://github.com/deepjavalibrary/djl-serving/actions/runs/6281012471/job/17058929176

**Review by lanking520 — APPROVED** [review]

**Pull request merged**